app/javascript/packages/i18n/index.spec.ts

```diff
@@ -45,5 +50,21 @@ describe('I18n', () => {
         expect(t('messages', { count: 2 })).to.equal('2 messages');
       });
     });
+
+    describe('array entry', () => {
+      context('with a singular key', () => {
+        it('returns array of strings', () => {
+          expect(t('list')).to.deep.equal(['one', 'two']);
+        });
+      });
+
+      context('with an array of the key', () => {
+        it('returns array of strings', () => {
+          const list = t(['list']).map((value) => value);
```

Comment:
For posterity, in case it raises eyebrows: The reason for the identity map is mostly to ensure that, when type-checked, TypeScript takes no issue with the fact that the return value would be expected to be an array, as an assurance of the contract that, when given an array of keys, the return value should be an array of messages.